extensions/blocks/membership-button/membership-button.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Memberships block.
+ *
+ * @since 7.3.0
+ *
+ * @package Jetpack
+ */
+
+if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+	require_once WP_CONTENT_DIR . '/mu-plugins/memberships/class-jetpack-memberships.php';
+} elseif ( Jetpack::is_active() ) {
+	require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
+}
+
+if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || Jetpack::is_active() ) {
```

Comment:
You shouldn't need to do this check since `is_active` is done before blocks registered with `jetpack_register_block`  are loaded:

https://github.com/Automattic/jetpack/blob/858b174e510ebcb59904cbeb0a89a072870d33e1/class.jetpack-gutenberg.php#L417-L430